packages/xstate-react/test/useInterpret.test.tsx

```diff
@@ -111,4 +111,28 @@ describe('useInterpret', () => {
 
     expect(actual).toEqual([1, 42]);
   });
+
+  it('should behave the same as `interpret` when initial context is not defined', (done) => {
+    const machine = createMachine({
+      initial: 'foo',
+      states: {
+        foo: {}
+      }
+    });
+
+    const interpretService = interpret(machine);
+
+    const App = () => {
+      const useInterpretService = useInterpret(machine);
+
+      expect(useInterpretService.initialState.context).toEqual(
```

Comment:
does the same issue happen for `useInterpretService.state.context`? If yes - then I would probably prefer to test that as using `initialState` is probably way less popular. Not even sure if we should expose `initialState` publicly on services, they are mostly useful for machines 🤔

Reply:
Thanks for this, I now know what you mean with `initialState` vs `state` vs `machine` and rewrote the tests for consistency in 29c2e33.